streaming/storage/extra.py

```diff
@@ -67,9 +35,46 @@ def _normalize_path(path: str) -> Tuple[str, bool]:
         path = obj.path
     else:
         is_local = False
+
+    path = os.path.normpath(path)
+
+    idx = path.find(':/')
```

Comment:
Does this logic also apply to DBFS and UC Volume file paths where their path is `dbfs:/Volumes/<catalog-name>/<schema-name>/<volume-name>/path`?

Reply:
How bout now?